app/uk/gov/hmrc/taxenrolmentassignmentfrontend/config/AppConfig.scala

```diff
@@ -33,6 +33,8 @@ class AppConfig @Inject()(val config: ServicesConfig) {
     : String = config.baseUrl("enrolment-store-proxy") + "/enrolment-store-proxy"
   lazy val TAX_ENROLMENTS_BASE_URL
     : String = config.baseUrl("tax-enrolments") + "/tax-enrolments"
+  lazy val AUTH_BASE_URL
+  : String = config.baseUrl("auth")
```

Comment:
Is this needed? we already get this as a part of the AuthConnector anyway.

Reply:
i thought this, but then, youd have to inject the AuthConnector which seems like a weird dependency 🤷